src/components/Filters/SchemaSieve.ts

```diff
@@ -124,44 +122,18 @@ export const createFullTextSearchFilter = (
 				(itemsAccumulator, entry) => {
 					const [key, item] = entry;
 					const model = getDataModel(item);
-					if (item.type?.includes('boolean') || !model) {
+					if (
+						item.type?.includes('boolean') ||
+						isDateTimeFormat(item.format) ||
+						!model
+					) {
 						return itemsAccumulator;
 					}
-					const defaultItem = {
+					itemsAccumulator.push({
 						key,
 						schema: item,
 						model,
-					};
-					const operators = model.operators(item);
-					if (
-						!item.type ||
-						!(item.type.includes('object') && isKeyValueObj(item))
-					) {
-						itemsAccumulator.push({
-							...defaultItem,
-							operator:
-								'contains' in operators
-									? { label: 'contains', slug: 'contains' }
-									: { label: 'is', slug: 'is' },
-						});
-					} else if (item.type?.includes('object') && isKeyValueObj(item)) {
-						itemsAccumulator.push({
-							...defaultItem,
-							operator: {
-								label: operators['key_contains' as keyof typeof operators],
-								slug: 'key_contains',
-							},
-							searchKey: findKey(item.properties!, { description: 'key' })!,
-						});
-						itemsAccumulator.push({
-							...defaultItem,
-							operator: {
-								label: operators['value_contains' as keyof typeof operators],
-								slug: 'value_contains',
-							},
-							searchKey: findKey(item.properties!, { description: 'value' })!,
-						});
-					}
```

Comment:
Did all this disappear b/c it's now handled by each case separately, rather than in a central way?

Reply:
Yes sir, I find it much more readable. I would keep it like this if you agree